static/src/javascripts/projects/common/modules/discussion/comments.js

```diff
@@ -0,0 +1,606 @@
+// @flow
+
+import $ from 'lib/$';
+import bean from 'bean';
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+import config from 'lib/config';
+import fetchJson from 'lib/fetch-json';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import { constructQuery } from 'lib/url';
+import Component from 'common/modules/component';
+import DiscussionApi from 'common/modules/discussion/api';
+import CommentBox from 'common/modules/discussion/comment-box';
+import WholeDiscussion from 'common/modules/discussion/whole-discussion';
+import relativedates from 'common/modules/ui/relativedates';
+import userPrefs from 'common/modules/user-prefs';
+import { inlineSvg } from 'common/views/svgs';
+
+type userType = {
```

Comment:
I think we should adopt the habit shared in many languages that types should be capitalised, to avoid confusion with variables or functions.